src/screens/OtherProfile/index.js

```diff
@@ -510,12 +603,11 @@ const OtherProfile = () => {
   };
 
   const handleRefresh = () => {
-    refetchFeeds(0);
+    getOtherFeeds(0);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getOtherFeeds` function is called when refreshing the feed. Ensure that this function correctly updates the state and UI to reflect the refreshed feed.
```diff
+ getOtherFeeds(0);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->